src/article.ts

```diff
@@ -112,16 +111,20 @@ export class Article {
         published: frontMatter.published,
       };
     }
-
-    if (remoteArticleBodyMarkdown && remoteArticleBodyMarkdown.trim() === body.body_markdown.trim()) {
+    if (remoteArticleBodyMarkdown && remoteArticleBodyMarkdown.trim() === body_markdown.trim()) {
       return {
         articleId: this.articleConfig.id,
         updateStatus: UpdateStatus.ALREADY_UP_TO_DATE as UpdateStatus.ALREADY_UP_TO_DATE,
         articleTitle: frontMatter.title,
         published: frontMatter.published,
       };
     }
-
+    const body = {
```

Comment:
Please define a new interface for this (and if possible add a comment above the interface with a link pointing to dev.to documentation :pray:) and type the `body` variable

Reply:
I don't use typescript sorry, so unsure how to do this correctly.